### ISSUE TITLE
LigandMPNN dataset 

### ISSUE THREAD

**Issue by bio-rat** [dauparas/LigandMPNN#3]
Hi,

Thank you for creating this awesome program. I have been using this on a crystal structure currently on PDB. I just want to know if it was in the training dataset. Where do I look up if my protein is in the dataset or not?

Have a good day!

**Comment by dauparas** [comment]
Hey!

Your PDB is likely in our training set if deposited before Dec 16, 2022, has a lower than 3.5 A resolution, and has a total number of residues in the PDB biounit smaller than 6000 residues. You can find details on the training data in the preprint (https://www.biorxiv.org/content/10.1101/2023.12.22.573103v1.full.pdf).

**Comment by bio-rat** [comment]
Hi, thank you for your response!

My PDB is 8DQH and it seems to fit the description of your training dataset. Is there a way I can make sure it is 100% in or not in the train or test set?
